guides/v2.1/cloud/project/projects.md

```diff
@@ -124,6 +125,13 @@ This displays the following page, which enables you to configure settings, [vari
 
 ![configure environments]({{ site.baseurl }}/common/images/cloud_project-conf-env.png)
 
+### Environment configuration variables
+On the Variables page, you can view, create, and manage environment variables for your project. For example, after we add your project to the {{ site.data.var.ece }} Fastly service account, you can view the Fastly API key and service ID credentials as shown in the following example.
+
+![Environment variables fastly credentials]({{ site.baseurl }}/common/images/cloud/cloud-project-web-ui-environment-variables.png)
+
+You can also view environment variables using the Magento CLI `magento-cloud variable:get -e <environment ID>` command. See [List and review variables]({{ page.baseurl }}/cloud/before/before-setup-env-2_clone.html#variablelist).
```

Comment:
Also splits the verb...
This command is long enough that it is difficult to see in the sentence. It might be more user friendly if they are able to copy it using the **Copy** function of a code block.